src/layman/layer/filesystem/input_file.py

```diff
@@ -110,6 +110,12 @@ def spatial_ref_crs_to_crs_id(spatial_ref):
     return crs_auth_name + ":" + crs_code
 
 
+def get_raster_crs_id(main_filepath):
+    in_data_source = gdal.Open(main_filepath)
+    crs = in_data_source.GetSpatialRef()
```

Comment:
Let's extract `get_raster_crs(main_filepath)` and use it in `get_raster_crs_id` and `check_raster_layer_crs`